### PR TITLE
Improve version clarity in About dialog

### PR DESCRIPTION
**Pull request by jmcphers**
### Intent

This change is intended to improve the clarity of the version information in the About dialog. 

Before:

image

After:

image

Addresses https://github.com/rstudio/rstudio/issues/9675. 
Addresses https://github.com/rstudio/rstudio/issues/9677. 

### Approach

- Tweak font sizes, weights, colors, and padding so that the most important part of the version, 2021-07.0 in this example, stands out the most.
- Add a "Pre-Release Build" banner to non-release builds to make it instantly obvious that they are not stable releases.
- Add a button to copy version information to the clipboard. 

### Automated Tests

N/A

### QA Notes

This change is pretty much style-only; the only new functionality here is the Copy Version button, which should ... copy the version. 

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

File: src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java [hunk 1]
```diff
@@ -87,6 +90,11 @@ public AboutDialogContents(ProductInfo info, ProductEditionInfo editionInfo)
             " is provided by RStudio, PBC for testing purposes only and is not an officially supported release."
          );
       }
+      else
+      {
+         preReleaseRibbon.setVisible(false);
```
Comment (MariaSemple) [inline src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java:95]: Is the pre-release ribbon visible by default?

**Review by MariaSemple — APPROVED** [review]
This is a great idea! LGTM!

File: src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java (hunk 1 above)
Comment (jmcphers) [reply src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java:95]: Yes, it just gets turned off in release builds. There's no visual difference either way, but if you think it'd make more sense I could have it hidden by default and then only shown in pre-release builds.

Comment (MariaSemple) [reply src/gwt/src/org/rstudio/studio/client/application/ui/AboutDialogContents.java:95]: Seems fine to me! I just couldn't see where it gets enabled, so I wanted to confirm my assumption.

**Pull request merged**